docs/proposals/InputDate.md

```diff
@@ -0,0 +1,62 @@
+# `InputDate`
```

Comment:
Should this have some type of acceptance criteria?

- When I type the date manually, the changes should reflect in the calendar interface?
- When I choose the date in the calendar interface, it should update the text field value
- It should handle date formats like 
  - MM/DD/YYYY
  - DD/MM/YYYY, 
  - Jan, DD, YYYY
  - etc.

Thoughts??

Reply:
Yea, should probably add that. I had sort of assumed that but it would be better to say it. :)